Source/JavaScriptCore/dfg/DFGLivenessAnalysisPhase.cpp

```diff
@@ -185,7 +185,7 @@ class LivenessAnalysisPhase : public Phase {
 
 } // anonymous namespace
 
-bool performLivenessAnalysis(Graph& graph)
+bool performGraphPackingAndLivenessAnalysis(Graph& graph)
 {
     graph.packNodeIndices();
```

Comment:
We should clear the FlowMap here.

Reply:
We should probably clear map only in debug build since it looks costly.

Reply:
> We should probably clear map only in debug build since it looks costly.

I was thinking in that way initially. But that wouldn't fix the problem I mentioned in the commit message if we only do that in debug mode, where the problem is untyped DFG nodes might be typed after packing since CFA only reset those typed  nodes.